### ISSUE TITLE
voiceOver reads time wrong

### ISSUE THREAD

**Issue by twinlensreflex** [bbc/simorgh#1579]
**Describe the bug**
voiceOver is reading time in timestamps as single digits eg "one five" instead of "one oh five" for `01:05`

apparently this is a bug with voiceOver

**To Reproduce**

1. Go to any article published today but 10 hours ago or more
2. Turn on voiceOver
3. navigate to timestamp
4. hear time digits read out incorrectly

**Expected behavior**
The time should be read out in a recognised format.

**Screenshots**
https://www.dropbox.com/s/4yl7x16vtph4q55/IMG_4061.MOV?dl=0 
video of behaviour

**Smartphone (please complete the following information):**

- Device: iPhone XS Max
- OS: iOS12
- Browser: Safari
- Version: 12.1

**Additional context**

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

**Comment by twinlensreflex** [comment]
https://docs.google.com/document/d/1ltd6kD1U0Gpix8iuKNd0a9u_zZKWBEiJRN7CuMBt4IM/edit Catharine has tested VoiceOver and written findings here

**Comment by greenc05** [comment]
Seems like that might of been one format I missed in my testing? https://docs.google.com/document/d/1ltd6kD1U0Gpix8iuKNd0a9u_zZKWBEiJRN7CuMBt4IM/edit

Though I guess that would make sense with the other bug I observed:

“The only exception seems to be with VoiceOver iPhone/Safari when the minutes start with a zero, such as ‘23:06’, in such cases it reads as ‘23:6’, while this isn’t ideal, I don’t think it’s worth the effort to resolve such a small use case”

I assume 01:15 is read as ‘one fifteen’ with safari/VoiceOver iOS @eleanombre? If that's the case I don't think we should try to resolve this, we should close this Issue and file a bug with all the details with Apple at accessibility@apple.com please @eleanombre

**Comment by twinlensreflex** [comment]
I've sent an email to accessibility@apple.com and am now waiting on a reply. This is blocked waiting on their response.

**Comment by twinlensreflex** [comment]
update from apple:

> Hello Eleanor,
>
> Thank you for writing to Apple. We appreciate you bringing this to our attention, and we have passed this report along to the appropriate team for investigation. We cannot comment on when it may be resolved.
>
> Sincerely,
> Apple Accessibility
